django_cloud_deploy/cli/new.py

```diff
@@ -76,6 +76,12 @@ def add_arguments(parser):
         action='store_true',
         help='Flag to indicate using a new or existing project.')
 
+    parser.add_argument(
```

Comment:
I think we need to add another prompt asking does the user want to deploy on GAE or GKE. But it is fine to implement this on another PR.

Reply:
i have this on the next pr coming up